<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UAV_Inertial</class>
 <widget class="QWidget" name="UAV_Inertial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCustomPlot" name="acceleration_plot" native="true"/>
     </item>
     <item>
      <widget class="QCustomPlot" name="velocity_plot" native="true"/>
     </item>
     <item>
      <widget class="QCustomPlot" name="position_plot" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="reset_inertial_frame">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="position_map" native="true"/>
     </item>
     <item>
      <widget class="Attitude" name="attitude" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Attitude</class>
   <extends>QWidget</extends>
   <header>Attitude.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
